.pre-commit-config.yaml

```diff
@@ -5,7 +5,8 @@ exclude: |
         InvenTree/InvenTree/static/.*|
         InvenTree/locale/.*|
         src/frontend/src/locales/.*|
-        .*/migrations/.*
+        .*/migrations/.* |
+        src/frontend/.yarn/.*
```

Comment:
can we also ignore the `yarn.lock` file? I recently had the following error:
```
codespell................................................................Failed
- hook id: codespell
- exit code: 65

src/frontend/yarn.lock:2152: OT ==> TO, OF, OR, NOT
```

Reply:
Probably